osal/src/os/posix/inc/os-freertos.h

```diff
@@ -40,21 +40,16 @@
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
-#include <limits.h>
 #include <signal.h>
 #include <time.h>
 #include <errno.h>
-#include <pthread.h>
-#include <mqueue.h>
-#include <fcntl.h>
-#include <semaphore.h>
 #include <sys/types.h>
-#include <sys/signal.h>
+#include <signal.h>
 
 /*
  * Use the global definitions from the shared layer
  */
-#include <os-shared-globaldefs.h>
+#include "os-shared-globaldefs.h"
```

Comment:
I don't think this should be altered

Reply:
Yeah I think its fine leaving it how it was. generally the quotation marks implies the header can be found within the project, which is what is happening in this case. but yeah can revert this

Reply:
yea that was my bad, I was playing around with header files and forgot to change it back.